src/http/request.ts

```diff
@@ -72,15 +72,24 @@ export class DrashRequest extends Request {
    * @param pathParams - The path params to match the request's URL to. The path
    * params come from a resource's path(s).
    * @param connInfo - The connection info Deno provides on a new request
+   * @param requestOptions - Any options to control the way requests behave.
    *
    * @returns A Drash request object.
    */
   static async create(
     request: Request,
     pathParams: Map<string, string>,
     connInfo: ConnInfo,
+    requestOptions?: RequestOptions,
```

Comment:
why not just `options`? 😛

Reply:
lmao probably coded too fast. the typing says request options and we're creating a request, so i think it should be `options` too

Reply:
doneski: https://github.com/drashland/drash/pull/671/commits/ec516042683eafafd6fbca0cfea1f59842e9b462